src/Questions.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Question from './Question';
+
+class Questions extends Component {
+  render() {
+    return (
+      <div>
```

Comment:
Are you sure you need to wrap <ul> with <div>?
Seems like it's working fine without additional wrapping.